v2.0/usage/configuration/as-service.md

```diff
@@ -83,7 +91,7 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  -v /run/docker/plugins:/run/docker/plugins \
  -v /lib/modules:/lib/modules \
  -v /var/run/calico:/var/run/calico \
- calico/node:v1.0.0-beta
+ calico/node:v1.0.0-rc2
```

Comment:
We haven't done this upgrade yet.

Everything should still say v1.0.0-beta until we cut the next RC.